ProjectTemplates/VisualStudio2010/Android/AndroidApplication1.csproj

```diff
@@ -21,6 +21,8 @@
     <TargetFrameworkVersion>v4.2</TargetFrameworkVersion>
     <MonoGamePlatform>Android</MonoGamePlatform>
     <MonoGameContentBuilderExe></MonoGameContentBuilderExe>
+    <MonoGameInstallDirectory Condition=" '$(OS)' != 'Unix' ">$(MSBuildProgramFiles32)</MonoGameInstallDirectory>
+    <MonoGameInstallDirectory Condition=" '$(OS)' == 'Unix' ">$(MSBuildExtensionsPath)</MonoGameInstallDirectory>    
```

Comment:
@tomspilman @KonajuGames are you guys happy with this change? The problem we are solving is that $(MSBuildProgramFiles32) does not exist on mac.. if the Windows created projects are to work on a mac too we need to point it to the $(MSBuildExtensionsPath) instead.. 

Alternatively move the Assemblies folder from Program Files\MonoGame\v3.0\Assembles to Program Files\MSBuild\MonoGame\v3.0\Assemblies then we could just use

$(MSBuildExtensionsPath)\MonoGame\v3.0\Assemblies\XXXX\MonoGame.Framework.dll